src/GraphQL/Execution/ExecutionContext.cs

```diff
@@ -11,6 +11,35 @@ namespace GraphQL.Execution
     public class ExecutionContext : IExecutionContext, IExecutionArrayPool, IDisposable
     {
 #pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
+        /// <summary>
+        /// Initializes a new instance.
+        /// </summary>
+        public ExecutionContext()
+        {
+        }
+
+        /// <summary>
+        /// Clones an existing instance, excluding <see cref="Errors"/>, <see cref="Metrics"/>, <see cref="OutputExtensions"/>,
```

Comment:
Why are these properties excluded? Let's write a reason.

Reply:
Probably something along the line of "as these properties hold result information"?

Reply:
Ah! Now I get it. Yes, something like that to emphasize attention that this ctor exists to copy **shared/reusable** state.

Reply:
New comment is this:
```cs
        /// <summary>
        /// Clones reusable state information from an existing instance; not any properties that
        /// hold result information. Specificially, <see cref="Errors"/>, <see cref="Metrics"/>,
        /// <see cref="OutputExtensions"/>, array pool reservations and internal reusable references
        /// are not cloned.
        /// </summary>
```

Reply:
👍